### PR TITLE
Modelzoo restructure breaks verifying user docker image

### PR DESCRIPTION
**Pull request by gongsu832**
Use docs/mnist_example/mnist.onnx for verifying user docker image so we don't depend on modelzoo, which has just been completely restructured.

This is a blocking issue for Jenkins CI so we should merge asap.

### CODE REVIEW

**Comment by gongsu832** [comment]
@jenkins-droid test this please

**Review by hamptonm1 — APPROVED** [review]
I am blocked until this PR is merged in, so approving since it is passing all checks.

**Pull request merged**
